### DISCUSSION TITLE
What is the function of `release_type` in the `/apps/upload`?

### DISCUSSION THREAD

**Discussion by rakuyoMo** [tryzealot/zealot#1734] [Q&A] [answered]
When uploading files using `/apps/upload`, no matter what the content of `release_type` is, the web page displays the result of the application unpacking, and no corresponding field is found in the web page.

![image](https://github.com/user-attachments/assets/ee3448bd-9411-4a80-8381-1d73b8b99351)

If the above image does not correspond to the `release_type` field, where is this field displayed? I can't find the corresponding location.

**Comment by rakuyoMo** [comment]
I found that when I upload using the web page and specify `release_type`, the position of the red box in the image is the value I filled in. But when I refresh the page, it is immediately overwritten by the unpacking information. Is this normal behavior? Is there a way to control it so that it will not be overwritten?

**Reply by icyleaf** [reply, marked as answer]
You are correct.

The release_type is a predefined constraint, and it is usually rarely customized by others. However, if a user uploads an application with this value, it should not be overridden by the app parser.

**Reply by icyleaf** [reply]
fixes in #1746
